test/sequential/test-heapdump-flag.js

```diff
@@ -0,0 +1,28 @@
+// Flags: --heapsnapshot-signal=SIGUSR2
+'use strict';
+const common = require('../common');
+
+if (!common.isMainThread)
+  common.skip('process.chdir is not available in Workers');
+
+const assert = require('assert');
+const fs = require('fs');
+const tmpdir = require('../common/tmpdir');
+
+assert.strictEqual(process.listenerCount('SIGUSR2'), 1);
+tmpdir.refresh();
+process.chdir(tmpdir.path);
+process.kill(process.pid, 'SIGUSR2');
```

Comment:
This won't work on Windows because uv_kill() only knows how to emulate a handful of signals and SIGUSR2 isn't one of them. I'd be okay with skipping this test on Windows.